Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -126,11 +126,13 @@ protected void LoadApplication(Application application)
 
 			application.PropertyChanged += AppOnPropertyChanged;
 
-			var iver = Platform.GetRenderer(application.MainPage);
-			if (iver != null)
+			if (application?.MainPage != null)
```

Comment:
The null check of application is unnecessary because it is already done on line 116.

Reply:
True. But we'll let it go anyway.